descriptors/ModuleDescriptor-template.json

```diff
@@ -1077,7 +1077,7 @@
     },
     {
       "id": "notes",
-      "version": "1.0"
+      "version": "2.0"
```

Comment:
Given that we think there are no changes which affect mod-circulation, I think it can provide dual support.